segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/stat/AutoScaleProcessorTest.java

```diff
@@ -116,17 +128,18 @@ public void writerCreationTest() throws Exception {
         AssertExtensions.assertEventuallyEquals(writerMock, () -> processor.getWriterFuture().join(), 10000L);
         AutoScaleEvent event = queue.take();
         assertEquals(event.getDirection(), AutoScaleEvent.DOWN);
-        
+
```

Comment:
revert

Reply:
Please revert all formatting changes to this class. It helps a lot with the signal-to-noise ratio for reviews.